python/engine/CMakeLists.txt

```diff
@@ -30,8 +30,8 @@ target_include_directories(pythonengine
 target_link_libraries(
   pythonengine
   PRIVATE
+  openstudiolib
```

Comment:
Yes we need this because of the Logger and yes Logger is part of the minimal utilities target, however minimal is a different compiled unit and since we want a single global Logger we cannot have it implmented in multiple compiled units. In fact, I think we can do away with the minimal target after this.